app/src/main/java/io/homeassistant/companion/android/widgets/StaticWidget.kt

```diff
@@ -158,7 +173,9 @@ class StaticWidget : AppWidgetProvider() {
                 appWidgetId,
                 entitySelection,
                 attributeSelection,
-                labelSelection
+                labelSelection,
+                textSizeSelection?.toInt() ?: 30,
```

Comment:
Should probably use `toIntOrNull()` so we don't throw and exception if it's not a number somehow.